custom_test.py

```
@@ -0,0 +1,128 @@
+    # Create smart orchestrator (automatically handles all agent registration)
+    orchestrator = SmartQADIOrchestrator()
+
+    # Ensure agents are ready
+    await orchestrator.ensure_agents_ready()
+
+    print("🤖 Smart orchestrator ready with automatic agent selection")
+
+    try:
+        # Run QADI cycle
+        print("🚀 Starting QADI cycle...")
+        start_time = datetime.now()
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)